lte/gateway/deploy/agw_install.sh

```diff
@@ -58,6 +67,21 @@ if [ "$(uname -r)" != "4.9.0-9-amd64" ]; then
   apt update
   # Installing prerequesites, Kvers, headers
   apt install -y sudo python-minimal aptitude linux-image-4.9.0-9-amd64 linux-headers-4.9.0-9-amd64
+  echo "Checking if kernel headers was installed, else install"
+  if ! [ "$(dpkg -l  | grep headers-4.9.0-9-amd64)" ]; then
+    # Get from alternative repository and add as debian packages local repo
+    wget http://cdimage.debian.org/mirror/cdimage/archive/9.9.0/amd64/iso-cd/debian-9.9.0-amd64-xfce-CD-1.iso
+    mkdir -p /mnt/disk
+    mount -o loop debian-9.9.0-amd64-xfce-CD-1.iso /mnt/disk
+    echo 'deb file:///mnt/disk/ stretch main contrib' >> /etc/apt/sources.list
+    sudo apt-get update --allow-unauthenticated
+    # Install kernel headers
+    sudo apt -y install linux-headers-4.9.0-9-amd64 --allow-unauthenticated
```

Comment:
should we also install linux-image-?

Reply:
Seems to come with the kernel installation. Magma installation works smoothly for me with this changes (it needs to be re-run though, but that's an unrelated issue I need to investigate and bring up)